### PR TITLE
[NUI] Update ViewAccessibility according to the coding rule

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- Updated ViewAccessibility code, such as description or coding style.
 1) ViewAccessibility.cs
 2) ViewAccessibilityEvent.cs 

### API Changes ###
- N/A

### CODE REVIEW

**Comment by dongsug-song** [comment]
@Seoyeon2Kim 
Could you update dali-csharp-binder's patch on this PR? and as you know, dali-csharp-binder patch should be merged prior to this PR. Otherwise there can be a runtime exception which blocks the TizenFX release by auto test hurdle.

Could you resolve the conflicts also?

**Comment by Seoyeon2Kim** [comment]
> @Seoyeon2Kim
> Could you update dali-csharp-binder's patch on this PR? and as you know, dali-csharp-binder patch should be merged prior to this PR. Otherwise there can be a runtime exception which blocks the TizenFX release by auto test hurdle.
> 
> Could you resolve the conflicts also?

As you can see all files of this PR, it's just to add and fix the description and the naming format according to C# coding style.
There's no dali-csharp-binder patch for this PR.
And I resolved the conflicts now.

**Review by lol-github — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs [hunk 1]
```diff
@@ -83,35 +125,66 @@ public bool Equals(GestureInfoType other)
...
     public class GestureInfoEventArgs : EventArgs
     {
+        /// <summary>
+        /// The gesture information type.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public GestureInfoType GestureInfo { get; internal set; }
+
+        /// <summary>
+        /// True if the event is consumed.
```
Comment (kimcinoo) [inline src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:161]: It seems that this information is for <returns> </returns>

Comment (dongsug-song) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:161]: @kimcinoo 
Hello, did you mean that `public int Consumed { get; set; }` has return value?
I think that this is a property so /<return> tag is not required.
how do you think?

Comment (kimcinoo) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:161]: OK this is description not only for "get"
This property type is "int", then we can use "True" for description?

**Review by hyunjushin — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:161]: Actually, this value is the second object of the template below in DALi. :
`using GesturePair = std::pair<Dali::Accessibility::GestureInfo, bool>;`
I'm not sure why the type is 'int' here, not 'bool' value.
I'll ask it to the person in charge and update it in another PR, for both type and description.

**Review by hinohie — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Comment by dongsug-song** [comment]
@Seoyeon2Kim 
Hello, 
I tried to do "Rebase and merge" but it had been inactivated with gray text.
please resolve the error below.

**_This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch._**

thank you.

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs (hunk 1 above)
Comment (grzelewski) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:161]: My interops were designed on SWIG examples were bool was converted to int before passing from c++ to c# layer. That's why this value is represented as int inside nui.

**Pull request merged**